BaseClasses.py

```diff
@@ -1090,7 +1094,8 @@ def __repr__(self):
         return self.__str__()
 
     def __str__(self):
-        return self.multiworld.get_name_string_for_object(self) if self.multiworld else f'{self.name} (Player {self.player})'
+        return self.multiworld.get_name_string_for_object(
+            self) if self.multiworld else f'{self.name} (Player {self.player})'
```

Comment:
Not sure how this happened. I did a squash commit for the first time ever since that's what someone wanted when I made the Wargroove 1 PR.